language/Records.md

```diff
@@ -79,6 +88,65 @@ A record update function can also be defined by using an `_` inplace of the reco
 _ { fieldName = newValue }
 ```
 
+## Record Puns
+
+_Record puns_ enable concise code when record fields have the same name as other values. This feature is useful for constructing records and pattern matching.
+
+### Constructing
+
+```purs
+origin :: Point
+origin = { x, y }
+-- origin = { x: x, y: y } -- Equivalent
+  where
+    x = 0.0
+    y = 0.0
+```
+
+### Pattern Matching
+
+```purs
+showPoint :: Point -> String
+showPoint { x, y } = show x <> ", " <> show y
+-- showPoint { x : x, y : y } = show x <> ", " <> show y -- Equivalent
+```
+
+### Not for Record Updates
+
+Note that puns may not be used for record updates.
+
+```purescript
+setX :: Number -> Point -> Point
+setX x point = point { x = x }
+-- setX x point = point { x } -- Not allowed
+```
+
+## Merging Records
```

Comment:
I'm glad to see the `record` library mentioned here. Perhaps this section could be named "Further record operations" and it could focus on merging but note that the `record` library provides other things too (for example, an efficient record builder via `Builder`).

Reply:
Applied suggestions of changing title and more explicitly linking to the `Record.Builder` module.